src/main/java/spleetwaise/address/storage/JsonAdaptedTag.java

```diff
@@ -9,7 +9,7 @@
 /**
  * Jackson-friendly version of {@link Tag}.
  */
-class JsonAdaptedTag {
+public class JsonAdaptedTag {
```

Comment:
This is because from what was told in class, there could possibly be bugs still within the address book. Therefore i decided to make this change as JsonAdaptedPerson uses this class with the following warning
![image](https://github.com/user-attachments/assets/e2996061-6162-4d70-b58a-8337cb9e2ea2)